src/printer.js

```diff
@@ -1801,14 +1801,20 @@ function printArgumentsList(path, options, print) {
     ]);
   }
 
+  // For next group, there are some exceptions where we don't want
+  // a trailing comma in the last comment.
+  const noTrailingCommaInLastComment = lastArg.type === "ObjectExpression";
```

Comment:
@vjeux if you don't mind, the best thing will be to close that PR as this is definitely not the correct approach to solving the problem 👍.